### PR TITLE
[SparseZoo Integration] update sparsezoo usage for 1.1 refactor

### PR DESCRIPTION
**Pull request by rahul-tuli**
This PR updates all usages of sparsezoo to conform with the latest sparsezoo_v2 upgrades as seen in SparseZoo (PR #179)[https://github.com/neuralmagic/sparsezoo/pull/179]

TODOs:

- Update non trivial usages of old `sparsezoo.objects.Model` class, tests with non-trivial usage also need to be updated

- Deprecate `sparsezoo.models` usage in favor of using stubs

### CODE REVIEW

File: src/deepsparse/transformers/helpers.py [hunk 1]
```diff
@@ -100,22 +101,25 @@ def get_onnx_path_and_configs(
...
-    elif require_configs and (config_path is None or tokenizer_path is None):
+        zoo_model = Model(model_path)
+        onnx_path = zoo_model.onnx_model.path
+        config_path = zoo_model.deployment.default.get_file(_MODEL_DIR_CONFIG_NAME).path
+        tokenizer_path = zoo_model.deployment.default.get_file(
+            _MODEL_DIR_TOKENIZER_NAME
+        ).path
+
+    else:
+        raise_value_error = True
```
Comment (bfineran) [inline src/deepsparse/transformers/helpers.py:112]: why is this logic change needed? taking a look

**Review by bfineran — CHANGES_REQUESTED** [review]
first review - will make suggested changes

**Comment by bfineran** [comment]
noting `test_engine`, `test_multi_engine`,  `test_examples`, and `test_custom_pipeline` will also need to be updated

**Comment by bfineran** [comment]
Pipelines from stubs passing manual local testing as well as `eval_downstream` with
```bash
deepsparse.transformers.eval_downstream zoo:nlp/sentiment_analysis/obert-base/pytorch/huggingface/sst2/pruned90-none --dataset sst2 --max-sequence-length 128
```

**Comment by bfineran** [comment]
verified stub input to `Engine` passes locally

**Comment by bfineran** [comment]
tests and src passing outside of `examples` and those tests now, opening for review.

will discuss early monday plans for examples

**Review by dbogunowicz — DISMISSED** [review]

**Review by bfineran — APPROVED** [review]

**Review by dbogunowicz — APPROVED** [review]

**Pull request merged**